dataset/src/main/scala/frameless/functions/UnaryFunctions.scala

```diff
@@ -28,6 +28,15 @@ trait UnaryFunctions {
     */
   def sortDescending[T, A: Ordering, V[_] : CatalystSortableCollection](column: TypedColumn[T, V[A]]): TypedColumn[T, V[A]] =
     new TypedColumn[T, V[A]](implicitly[CatalystSortableCollection[V]].sortOp(column.untyped, sortAscending = false))(column.uencoder)
+
+
+  /** Creates a new row for each element in the given collection. The collection needs to implement the CatalystExplodableCollection
```

Comment:
> The collection needs to implement the CatalystExplodableCollection type class.

The formulation seems odd since user won't be able to implement this type class to make it work... What about "The column types eligible for this operation is constrained by the CatalystExplodableCollection"?